TELCODOCS-1850: Delay Garbage Collection for successful builds / signing jobs (#76669)
openshift/openshift-docs: modules/kmm-configuring-kmmo.adoc

| `healthProbeBindAddress`
| Defines the address on which the Operator monitors for kubelet health probes. The recommended value is `:8081`.

|`job.gcDelay`

> |Defines the duration for which successful build pods should be preserved before they are deleted. There is no recommended value for this setting. For information on the valid values for this setting, see link:https://pkg.go.dev/time#ParseDuration[ParseDuration].

Reviewer: * Per ISG, should be "information about"
* Per ISG, you don't have to reword to use the "for which" construction if it makes the sentence read a bit awkwardly. Recommend rewording
* And just a question if it's necessary to say "There is no recommended value for this setting." I know you put recommended values in the others, but I would assume that saying nothing would mean the same thing and would be more concise. But also fine if you feel like it's necessary.

```suggestion
|Defines the duration that successful build pods should be preserved for before they are deleted. There is no recommended value for this setting. For information about the valid values for this setting, see link:https://pkg.go.dev/time#ParseDuration[ParseDuration].
```

Author: Made these changes.

|`worker.setFirmwareClassPath`
|Sets the kernel's firmware search path into the `/sys/module/firmware_class/parameters/path` file on the node. The recommended value is `/var/lib/firmware` if you need to set that value through the worker app. Otherwise, unset.

> |`job.gcDelay`

Author: @qbarrand Please verify this is the correct placement for this. Also, I think it needs to be added to the example file above the table.

Reviewer 2: Could we follow the alphabetical order for those settings?
Agreed, this could be added to the example above:
```yaml
job:
  gcDelay: 1h
```

Author: Please check new placement in the example and the table.

Reviewer 2: Looks good to me.
